dev-libs/libtpms/libtpms-0.6.0.ebuild

```diff
@@ -0,0 +1,27 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit autotools eutils
```

Comment:
Drop `eutils` since you're not using anything from it.

Reply:
Dropped